pkgs/tools/misc/nvimpager/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, pkgs
+# Defaults to false until I find out how to provide /dev/tty inside the
+# sandbox.
+, doCheck ? false
```

Comment:
It's fine to just set it to `false` for now. Also, declaring it here as parameter is not necessary since `doCheck` can be modified using `overrideAttrs`.